test.js

```diff
@@ -45,25 +45,41 @@ test('createQuad', t => {
       positions: [[1, -3, 0], [1, 3, 0], [-1, 3, 0], [-1, -3, 0]],
       normals: [[0, 0, -1], [0, 0, -1], [0, 0, -1], [0, 0, -1]],
       cells: [[3, 2, 1, 0]]
-    }, "A quad simplicial complex is created")
+    }, 'A quad simplicial complex is created')
 
-    t.deepLooseEqual(cell, [3, 2, 1, 0], "The cell is also returned.")
+    t.deepLooseEqual(cell, [3, 2, 1, 0], 'The cell is also returned.')
   })
 
   test('createQuad normal facing is correct', t => {
     t.plan(6)
     const xp = quad.createQuad({facing: 'x+'})
-    t.deepLooseEqual(xp.quads.normals[0], [1, 0, 0], "x+ points up")
+    t.deepLooseEqual(xp.quads.normals[0], [1, 0, 0], 'x+ points up')
     const xm = quad.createQuad({facing: 'x-'})
-    t.deepLooseEqual(xm.quads.normals[0], [-1, 0, 0], "x- points down")
+    t.deepLooseEqual(xm.quads.normals[0], [-1, 0, 0], 'x- points down')
     const yp = quad.createQuad({facing: 'y+'})
-    t.deepLooseEqual(yp.quads.normals[0], [0, 1, 0], "y+ points up")
+    t.deepLooseEqual(yp.quads.normals[0], [0, 1, 0], 'y+ points up')
     const ym = quad.createQuad({facing: 'y-'})
-    t.deepLooseEqual(ym.quads.normals[0], [0, -1, 0], "y- points down")
+    t.deepLooseEqual(ym.quads.normals[0], [0, -1, 0], 'y- points down')
     const zp = quad.createQuad({facing: 'z+'})
-    t.deepLooseEqual(zp.quads.normals[0], [0, 0, 1], "z+ points up")
+    t.deepLooseEqual(zp.quads.normals[0], [0, 0, 1], 'z+ points up')
     const zm = quad.createQuad({facing: 'z-'})
-    t.deepLooseEqual(zm.quads.normals[0], [0, 0, -1], "z- points down")
+    t.deepLooseEqual(zm.quads.normals[0], [0, 0, -1], 'z- points down')
   })
+
+  test('fromFunction creates a grid of quads', t => {
+    //
+    // ^ v
+    // |
+    // 6  7
+    // 3  4
+    // 0  1  --> u
```

Comment:
Realized very quickly this needs careful consideration of orientation. Did this in raster order by accident, and then backface culling was backwards because of the way I oriented my quads.